### DISCUSSION TITLE
Get 🌟 StartStruck

### DISCUSSION THREAD

**Discussion by abhinit21** [Schweinepriester/github-profile-achievements#451] [Q&A]
**Repo** - https://github.com/abhinit21/abhinit21
😊Thanks !

_Provide any repo you want to receive Star in reply_

**Comment by exkludera** [comment]
https://github.com/70L/70L 😊

**Reply by montasim** [reply]
Done @70L :
Do it for me, please.
https://github.com/montasim/montasim

**Reply by d4tdev** [reply]
@montasim 
https://github.com/d4rtj/d4rtj
done, help me

**Reply by montasim** [reply]
Done @d4rtj

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by montasim** [reply]
Done @ArielMAJ

**Reply by ArielMAJ** [reply]
Ty

**Comment by d4tdev** [comment]
done
https://github.com/d4rtj/d4rtj
help me !

**Reply by abhinit21** [reply]
Done @d4rtj

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by om3ga6400** [comment]
@abhinit21  https://github.com/OMEGA6405/OMEGA6405

**Reply by abhinit21** [reply]
Done 👍
